cmd/config.go

```diff
@@ -23,6 +22,7 @@ func configCmd(ctx *config.Context) *cobra.Command {
 	cmd.AddCommand(
 		configShowCmd(ctx),
 		configInitCmd(),
+		configEditCmd(ctx),
```

Comment:
Shouldn't you add this function not to the `config` subcommand but to the `paths` subcommand?

Reply:
move to the paths subcommand
https://github.com/hyperledger-labs/yui-relayer/pull/97/commits/e39fd4ca6b0bca41784ea3b9599a29944feef875